### PR TITLE
Cheat sheet template: Added missing template types

### PR DESCRIPTION
**Pull request by tagawa**
Add put them in alphabetical order.

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

File: template/share/goodie/cheat_sheets/json/example.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "first alias", "second alias", "etc"
     ],
 
-    "template_type": "one of: keyboard, terminal, code, reference",
+    "template_type": "one of: code, keyboard, language, link, reference, terminal",
```
Comment (GuiltyDolphin) [inline template/share/goodie/cheat_sheets/json/example.json]: How about `reference, or terminal`?

**Comment by GuiltyDolphin** [comment]
@tagawa Ignore my comment - I thought this was for docs :wink:

Looks good to me too! :dolphin:

**Pull request merged**